ddtrace/contrib/pytest/newhooks.py

```diff
@@ -0,0 +1,29 @@
+"""pytest-ddtrace hooks.
```

Comment:
Including the word "new" in the module name raises the question of whether there are also "old" hooks somewhere that have maybe been deprecated. If there is a clear reason to use "new" here, I recommend stating it in the docstring. If not, I'd suggest removing the "new" naming.

Reply:
I followed `pytest-xdist`'s naming scheme (and that's the plugin Pytest's own docs point to for "writing new hooks" in [their docs](https://docs.pytest.org/en/7.1.x/how-to/writing_hook_functions.html#declaring-new-hooks) . It's somewhat consistent with [other plugins](https://github.com/search?q=path%3A**%2Fnewhooks.py+NOT+xdist&type=code).

I think the point is that they're new hooks being introduced to `pytest` by the plugin. FWIW, I also found it a bit confusing, and followed more because it looked like convention than anything.